core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -144,6 +144,11 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
       ThreadUtils.newDaemonSingleThreadScheduledExecutor("cleanup-decommission-execs")
     }
 
+  override def getDriverLogUrls: Option[Map[String, String]] = {
```

Comment:
Instead of here, please override this at `KubernetesClusterSchedulerBackend` instead of `CoarseGrainedSchedulerBackend` because this PR aims for K8s only.
> For now, only K8s cluster manager supports this configuration.

Reply:
moved